### PR TITLE
adds travis

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+elixir:
+  - 1.4.5
+  - 1.5.1
+otp_release:
+  - 19.3
+  - 20.0
+services:
+  - docker
+env:
+  - GOOGLE_APPLICATION_CREDENTIALS=$HOME/credentials.json
```
Comment (dazuma) [inline .travis.yml]: Is the project set in the travis config?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+language: elixir
+elixir:
+  - 1.4.5
+  - 1.5.1
```
Comment (dazuma) [inline .travis.yml]: 1.5.2

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+otp_release:
+  - 19.3
+  - 20.0
```
Comment (dazuma) [inline .travis.yml]: 20.1

File: .travis.yml (hunk 2 above)
Comment (bshaffer) [reply .travis.yml]: How about we just cover the most current release then?

File: .travis.yml (hunk 3 above)
Comment (bshaffer) [reply .travis.yml]: How about we just cover the most current release then? e.g.

 otp_release:
   - 19
   - 20
```

File: .travis.yml (hunk 2 above)
Comment (dazuma) [reply .travis.yml]: sgtm

File: .travis.yml (hunk 3 above)
Comment (dazuma) [reply .travis.yml]: sgtm

File: .travis.yml (hunk 1 above)
Comment (bshaffer) [reply .travis.yml]: Yes! In the `GOOGLE_PROJECT_ID` environment variable. We *could* set this here as well, but I figure this is more straightforward.

File: .travis.yml (hunk 3 above)
Comment (bshaffer) [reply .travis.yml]: Apparently you can't do that, so I added the numbers back 😄

**Comment by bshaffer** [comment]
Thanks @dazuma !

**Pull request merged**